<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>870</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>870</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="OponField">
    <property name="geometry">
     <rect>
      <x>436</x>
      <y>-5</y>
      <width>401</width>
      <height>411</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../build-SeaBattle-Desktop_Qt_5_12_3_MinGW_64_bit-Debug/debug/qmake_qmake_immediate.qrc">:/Images/real_field.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>450</y>
      <width>261</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QLabel" name="myField">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>401</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../build-SeaBattle-Desktop_Qt_5_12_3_MinGW_64_bit-Debug/debug/qmake_qmake_immediate.qrc">:/Images/real_field.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Go">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>450</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>GO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Restart">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>450</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Restart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PlayAIButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>261</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Play With AI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ServerButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>480</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect to server</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>410</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="FindOpon">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>480</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Players List</string>
    </property>
   </widget>
   <widget class="QLabel" name="enemy_nameLabel">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>420</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="turnLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>520</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../build-SeaBattle-Desktop_Qt_5_12_3_MinGW_64_bit-Debug/debug/qmake_qmake_immediate.qrc"/>
 </resources>
 <connections/>
</ui>
